src/coreclr/vm/interpreter.cpp

```diff
@@ -882,6 +898,10 @@ CorJitResult Interpreter::GenerateInterpreterStub(CEEInfo* comp,
         // x8 through x15 are scratch registers on ARM64.
         IntReg x8 = IntReg(8);
         IntReg x9 = IntReg(9);
```

Comment:
All these are local vaues and are unused codes.
So the LoongArch64 is just a noting and is ok to be deleted.